utils/sql-db-seeder/src/utilisation-report/utilisation-report.seed.ts

```diff
@@ -27,6 +27,10 @@ export default class UtilisationReportSeeder implements Seeder {
     const uploadedReportReportPeriod = getCurrentReportPeriodForBankSchedule(paymentReportOfficerBank.utilisationReportPeriodSchedule);
     const uploadedReport = createUploadedReport(paymentReportOfficer, uploadedReportReportPeriod, 'PENDING_RECONCILIATION');
 
+    // The reports need to be seeded either before or with the fee records
+    const { feeRecordWithMatchingCurrencies, feeRecordWithDifferingCurrencies } = createFeeRecordsForReport();
+    uploadedReport.feeRecords = [feeRecordWithMatchingCurrencies, feeRecordWithDifferingCurrencies];
```

Comment:
`uploadedReport.updateWithFeeRecords(feeRecords: createFeeRecordsForReport())` maybe?

Reply:
In actual production code we almost never interact with properties directly and use more of this object-oriented style. I didn't mind as much here because it is just for seeding, but I suppose for consistency's sake we should use that method